demo/accel-config-demo/test_runner_disable_shared_queues.patch

```diff
@@ -282,11 +290,59 @@
 +for opcode in "0x4d" "0x49" "0x4c" "0x48" "0x43"; do
  	test_op $opcode $flag
  done
- 
-@@ -261,7 +259,3 @@
- flag="0x0"
- test_op_filter $flag
- 
+-
+-if [ "$IDXD_VERSION" != "0x100" ]; then
+-	flag="0x1"
+-	echo "Testing with 'block on fault' flag ON"
+-	for opcode in "0x4e" "0x4a"; do
+-		test_op $opcode $flag
+-	done
+-
+-	flag="0x0"
+-	echo "Testing with 'block on fault' flag OFF"
+-	for opcode in "0x4e" "0x4a"; do
+-		test_op $opcode $flag
+-	done
+-fi
+-
+-flag="0x1"
+-test_op_filter $flag
+-
+-flag="0x0"
+-test_op_filter $flag
+-
+-if [ "$IDXD_VERSION" != "0x100" ]; then
+-	flag="0x1"
+-	aecs_flag="0x0101"
+-	echo "Testing with 'block on fault' flag ON"
+-	for opcode in "0x41" "0x40"; do
+-		test_op_crypto $opcode $flag $aecs_flag
+-	done
+-
+-	flag="0x0"
+-	aecs_flag="0x0301"
+-	echo "Testing with 'block on fault' flag OFF"
+-	for opcode in "0x41" "0x40"; do
+-		test_op_crypto $opcode $flag $aecs_flag
+-	done
+-
+-	bind_vfio
+-
+-	flag="0x1"
+-	echo "Testing with 'block on fault' flag ON"
+-	for opcode in "0x0a"; do
+-		test_op_transl_fetch $opcode $flag
+-	done
+-
+-	flag="0x0"
+-	echo "Testing with 'block on fault' flag OFF"
+-	for opcode in "0x0a"; do
+-		test_op_transl_fetch $opcode $flag
+-	done
+-
+-	unbind_vfio
+-fi
+-
```

Comment:
How much does this decrease test coverage?

Reply:
@tkatila 
Coverage wise it's identical to the previous version, i.e. these tests are new in 4.0.